theme-utils.mjs

```diff
@@ -1349,6 +1361,102 @@ async function escapePatterns() {
 	}
 }
 
+async function validateThemes( themes ) {
+	function readJson( file ) {
+		return fs.promises.readFile( file, 'utf-8' ).then( JSON.parse );
+	}
+	async function loadSchema( uri ) {
+		if ( ! uri || ! URL.canParse( uri ) ) {
```

Comment:
In core we're using node `>=20.10.0`, so it would be nice to match that. The problem is that `@es-joy/jsdoccomment@0.36.1` doesn't support node 20, so `@wordpress/scripts` needs to be updated. I'll open a separate PR for the node version bump since it involves a few changes.

```
themes@1.0.564
└─┬ @wordpress/scripts@26.4.0
  └─┬ @wordpress/eslint-plugin@14.6.0
    └─┬ eslint-plugin-jsdoc@39.9.1
      └── @es-joy/jsdoccomment@0.36.1
```

Reply:
Opened #8045 to fix it.